config/root-config.in

```diff
@@ -146,7 +145,7 @@ fi
 ### machine dependent settings ###
 
 # Set the C++ standard version
-cxxversionflag="-std=${cxxversion} "
+cxxversionflag="-std=c++@CMAKE_CXX_STANDARD@ "
```

Comment:
Is that really correct?  (i.e. Isn't CMAKE_CXX_STANDARD=17 requires -std=c++1z on some compiler?)

Reply:
Okay ... are we saying that we will not support any compiler/standard pair that is not fully numerical? (i.e. so no 20 support until the compiler officially use --std=c++20)?  [That is a valid choice that ought to be 'documented' somewhere and imply that we do not support experimenting with upcoming standard, only established ones ... or am I missing something]

Reply:
No, but that's the status quo. We only support C++11, 14, and 17. Support for C++20 will require more work than just adding the flag in root-config, as was the case for C++17 due to removed things from older C++ standards, etc.